backend/django/project/settings.py

```diff
@@ -105,7 +109,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'America/New_York'
```

Comment:
I would recommend leaving this as UTC, unless there's some other compelling reason to change it. It makes parsing logs easier and also makes timezone conversions more straightforward when needed, as you don't need to worry about the offset.